tests/core/utils/test_linker_utils.py

```diff
@@ -1,6 +1,6 @@
 from eth_utils import remove_0x_prefix, to_hex
 from eth_utils.toolz import assoc
-from ethpm.utils.chains import create_block_uri, get_genesis_block_hash
+from ethpm._utils.chains import create_block_uri, get_genesis_block_hash
```

Comment:
still bad mojo here, you should just vendor these utilities rather than maintaining the utils imports.  Otherwise you're very likely to break things again.